requirements.txt

```diff
@@ -36,3 +36,4 @@ unittest-xml-reporting==2.5.1
 vcrpy==2.0.1
 vcrpy-unittest==0.1.7
 whitenoise[brotli]==4.1.2
+conllu
```

Comment:
which version?

Reply:
Should this be pinned to `conllu==1.3.1`?

Reply:
Yes, is 1.3.1